addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/users-controller.js

```diff
@@ -74,22 +74,23 @@ async function configure(context) {
     }),
   );
 
+  // TODO: Do we want to support changing user passwords? Currently this API is only supported for internal Auth
   // ===============================================================
   //  PUT /:username/password (mounted to /api/users)
   //  In this case it is relevant to identify user by username/authProvider
   // ===============================================================
-  router.put(
-    '/:username/password',
-    wrap(async (req, res) => {
-      const requestContext = res.locals.requestContext;
-      const username = req.params.username;
-      const { password } = req.body;
-
-      // Save password salted hash for the user in internal auth provider (i.e., in passwords table)
-      await dbPasswordService.savePassword(requestContext, { username, password });
-      res.status(200).json({ username, message: `Password successfully updated for user ${username}` });
-    }),
-  );
+  // router.put(
```

Comment:
Could we also update API changes in the openapi file?

Reply:
good idea. Updated OpenApi file.